lib/web_of_science/client.rb

```diff
@@ -9,11 +9,12 @@ module WebOfScience
   # http://ipscience-help.thomsonreuters.com/wosWebServicesExpanded/WebServicesExpandedOverviewGroup/Introduction.html
   # It uses the savon gem for SOAP, see http://savonrb.com/version2/client.html
   class Client
-
     API_VERSION = '3.0'.freeze # Based on USER GUIDE July 7, 2015
     AUTH_WSDL = 'http://search.webofknowledge.com/esti/wokmws/ws/WOKMWSAuthenticate?wsdl'.freeze
     SEARCH_WSDL = 'http://search.webofknowledge.com/esti/wokmws/ws/WokSearch?wsdl'.freeze
 
+    delegate :logger, to: :WebOfScience
+
```

Comment:
Does this require including the Forwardable module?

Reply:
No.  Quite annoyingly it is a slightly different syntax than `Forwardable`.  I used the `Forwardable` syntax in classes that already included it.  I used the stock syntax everywhere else.